packages/host/tests/acceptance/operator-mode-test.ts

```diff
@@ -1250,7 +1253,7 @@ module('Acceptance | operator mode tests', function (hooks) {
   });
 
   test<TestContextWithSave>('unsaved changes made in card editor are saved when switching out of code mode', async function (assert) {
-    assert.expect(2); // the test waiters permit a normal auto-save to sneak in. the 2nd save is the save on close.
+    assert.expect(1);
```

Comment:
the 500ms delay for making things human readable slowed down the auto-save that was sneaking in :tada: